README.md

```diff
@@ -63,8 +409,9 @@ query {
 }
 ```
 
-we should see this result 
-```
+The result should give information about three gravatars:
```

Comment:
thanks!! All looks good to me, just a small thing that might worth mentioning: in the deploy contract step, there is also a "create gravatars" migration, https://github.com/AcalaNetwork/subgraph-example/blob/master/migrations/3_create_gravatars.js. This is where a data comes from, otherwise ppl might confused why there is already some gravitar data after deploying the contract